enterprise/migrations/0001_squashed_0092_auto_20200312_1650.py

```diff
@@ -138,19 +136,6 @@ class Migration(migrations.Migration):
                 'ordering': ['created'],
             },
         ),
-        migrations.CreateModel(
```

Comment:
This is related to the disappearing of the `AddField` line. Tests were complaining that they couldn't find this field through the manager, and adding it to the definition fixed it. Because the new field is a foreign key to another model defined later in the file, I had to move it lower.